packages/components/psammead-copyright/src/index.stories.jsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import notes from '../README.md';
 import Copyright from './index';
 
 storiesOf('Copyright', module)
-  .add('default', () => <Copyright>Getty Images</Copyright>, { notes })
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    () => <Copyright>{text('copyright', 'Getty Images')}</Copyright>,
+    { notes, knobs: { escapeHTML: false } },
+  )
   .add(
     'with visually hidden text',
     () => (
       <Copyright>
-        <VisuallyHiddenText>Image source, </VisuallyHiddenText>
-        Getty Images
+        <VisuallyHiddenText>{text('Image source,')} </VisuallyHiddenText>
```

Comment:
I think this needs a default value of `Image source,` currently it runs as blank